pymc4/coroutine_model.py

```diff
@@ -91,9 +93,9 @@ def __call__(
 
         Parameters
         ----------
-        name : str
-            The desired name for the model, by default, it is inferred from the model declaration context,
-            but can be used just once
+        name : str or ``_no_name_provided``
```

Comment:
Could you add an example of the use of the `args` parameter?

Reply:
Would you mind if I add issues to capture your two questions and defer these an issue and then another PR? I personally don't know but am working through understanding PyMC4 by asking Maxim a bunch of questions?

Reply:
I would not mind at all, that sounds like a great idea.